mesos-simulation/src/test/scala/mesosphere/mesos/scale/MetricsResult.scala

```diff
@@ -0,0 +1,160 @@
+package mesosphere.mesos.scale
+
+import play.api.libs.json._
+import org.apache.commons.io.FileUtils
+import java.io.File
+
+sealed trait Metric {
```

Comment:
It might make sense to rewrite DisplayAppScaling result to use this infrastructure.

It feels weird to have two approaches in the same package.